homeworks/taron_hakobyan/Project2.java

```diff
@@ -0,0 +1,23 @@
+public class Project2{
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		int number = scanner.nextInt();
+        String[] flowers = new String[31];
+        for (int i = 0; i < flowers.length; ++i) {
+            flowers[i] = "flower" + String.valueOf(i);
+        }
+        getFlowers(flowers,number);
+    }
+
+    static void getFlowers(String flowers[], int number) {
+        int[] binaryLook = new int[31];
+        for (int i = 30; i >= 0; i++) {
```

Comment:
weird looking for, did you test the code? you will get out of bound exception at the second iteration